server/service/IndividualServicesService.js

```diff
@@ -474,7 +474,7 @@ exports.regardApplication = async function (body, user, originator, xCorrelator,
       let headers = { user, xCorrelator, traceIndicator, customerJourney, lengthOftheForwarding }
       let Result = await RegardApplication.RegardapplicationUpdate(applicationName, releaseNumber, headers);
       var response = {};
-      if (Result.success) {
+      if (Result.successConnected) {
```

Comment:
SuccessConnected shall be successfullyConnected. Please change this occurrence in all the places.